dcmorales/digital-keyboard-tunes
# Feature: play selected notes

## Description
Builds on the play functionality. A basic play button has been added. When it is pressed, it will play a series of notes based on the selected keyboard settings. Currently there are only two options for 'scale': 'chromatic' and 'major'. Only chromatic is functional. All other scales will be handled as a different feature. 
The context has been updated to provide an activeNote as well. This applied a basic 'active' type class to the key so that the text changes color. This will be handled in more detail as another feature. 

## Steps to test / reproduce
From home page, update selections in the settings. Press the play button. It should play 13 keys that start with the selectedKey at the selectedOctave. The text in each key from that octave should change color.

## Types of changes



- [ ] Task (non-breaking change that is not a bug or feature)

## Reminders



- [x] I have tested that my code builds successfully
- [x] I have pulled the latest version of this branch's base branch (i.e. `main`)
- [x] I have updated the README or inline docs if needed
- [x] I have added a short, descriptive title to this PR

## Other comments
